packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -252,6 +259,7 @@ export class ColorPickerBase extends React.Component<IColorPickerProps, IColorPi
                         spellCheck={false}
                         ariaLabel={textLabels[comp]}
                         autoComplete="off"
+                        {...(comp !== 'hex' ? getRgbaAriaProps(comp) : null)} // We don't need aria props for hex
```

Comment:
Nit, you can do this instead and the spread operator will behave the same (we're generally trying to avoid explicit null in new code)
```suggestion
                        {...(comp !== 'hex' && getRgbaAriaProps(comp))} // We don't need aria props for hex
```